lisa/features/disk_ephemeral.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.feature import Feature
```

Comment:
Because those features are similar, and the pattern of Python asks to reduce file numbers. So they can be put into the same file, like disks.py